specification/imds/data-plane/Microsoft.InstanceMetadataService/stable/2018-10-01/imds.json

```diff
@@ -0,0 +1,760 @@
+{   
+    "swagger": "2.0",
+    "info": {
+        "title": "InstanceMetadataClient",
+        "description": "The Azure Instance Metadata Client",
+        "version": "2018-10-01"
+    },
+    "host": "169.254.169.254",
+    "basePath": "/metadata",
+    "schemes": [
+        "http"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "security": [
+        {
+            "azure_auth": [
+                "user_impersonation"
+            ]
+        }
+    ],
+    "securityDefinitions": {
+        "azure_auth": {
+            "type": "oauth2",
+            "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+            "flow": "implicit",
+            "description": "Azure Active Directory OAuth2 Flow",
+            "scopes": {
+                "user_impersonation": "impersonate your user account"
+            }
+        }
+    },
+    "paths": {
+        "/instance": {
+            "get": {
+                "operationId": "Instance_GetMetadata",
```

Comment:
This should be Instances_GetMetadata

> OperationId has a noun that conflicts with one of the model names in definitions section. The model name will be disambiguated to 'InstanceModel'. Consider using the plural form of 'Instance' to avoid this. Note: If you have already shipped an SDK on top of this spec, fixing this warning may introduce a breaking change.